src/main/kotlin/kotlinx/kover/api/KoverConfig.kt

```diff
@@ -296,6 +312,15 @@ public open class KoverMergedFilters @Inject constructor(private val objects: Ob
         classes.set(classFilter)
     }
 
+    /**
+     * Configures annotation filter.
```

Comment:
Please use consistent approach to the documentation and thoroughly document each public declaration: what are these filters and what for. Like you've done for `annotations` block above